### ISSUE TITLE
508-defect-1: Groups of form elements must have group labels (e.g. <fieldset> and <legend>)

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/va.gov-team#55379]
### Point of contact

Jeana

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

Groups of form elements MUST have group labels (e.g. `<fieldset>` and `<legend>`, to meet Section 508 requirements. The current code in the Statement of Truth wraps the text in `<p>`  Instead, there should be a `<fieldset>` within the statement of truth that wraps around the text and the form fields. `<legend>` would be wrapped around the two paragraphs before the name field.

### Reproduction steps

1. Enter `https://staging.va.gov/housing-assistance/disability-housing-grants/adapted-housing/` in a browser
2. Continue through the form until you get to the review page
3. Start screen reader and tab to the statement of truth
4. Verify that there isn't a semantic correlation between "I certify that I am applying for assistance..." and the name field and checkbox that follows.

### Proposed solution or next steps

```
<article class="vads-u-background-color--gray-lightest vads-u-padding-bottom--3 vads-u-padding-x--3 vads-u-padding-top--1px vads-u-margin-bottom--3">
    <fieldset>
        <legend>
            <h3>Statement of truth</h3>
        </legend>
        <div id="certify-text">
            <p>I certify that I am applying for assistance in acquiring specially adapted housing or special home adaptation grant because of the nature of my service-connected disability. I understand that there are medical and economic features yet to be considered before I am eligible for this benefit, and that I will be notified of the action taken on this application as soon as possible.</p>
            <p>I have read and accept the <a aria-label="Privacy policy, will open in new tab" target="_blank" href="/privacy-policy/">privacy policy</a>.</p>
        </div>
        <div aria-labelledby="certify-text">
            <va-text-input label="Your full name" class="signature-input hydrated" id="veteran-signature" name="veteran-signature" type="text" required="true" error=""></va-text-input>
            <div class="form-checkbox">
                <input id="errorable-checkbox-1" name="veteran-certify" type="checkbox">
                <label name="veteran-certify-label" for="errorable-checkbox-1">I certify the information above is correct and true to the best of my knowledge and belief.<span class="form-required-span">(*Required)</span>
                </label>
            </div>
        </div>
    </fieldset>
</article>
```

### References, articles, or WCAG support

[WCAG 2.0 Level A - 1.3.1 - Info and Relationships: Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

[Wuhcag 1.3.1 – Info and Relationships (Level A)](https://www.wuhcag.com/info-and-relationships/)

[WCAG 2.0 Level A - 4.1.1 - Parsing](http://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html)

### Type of issue

- [X] Screenreader
- [X] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [X] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by jeana-adhoc** [comment]
There is an additional issue with the statement of truth box and certification after it follows this recommended revision:
Screen reader users are forced to listen to the entire legalese and aren't given an option whether or not they want to consume it. Whereas sighted users can choose to just scan the text.
